webui/react/src/utils/browser.ts

```diff
@@ -0,0 +1,7 @@
+export const updateFavicon = (iconPath: string): void => {
+  const linkEl: HTMLLinkElement | null = document.querySelector("link[rel*='shortcut icon']");
+  if (!linkEl) return;
+  linkEl.type = 'image/x-icon';
+  linkEl.rel = 'shortcut icon';
+  linkEl.href = iconPath;
+};
```

Comment:
abstract the path and keeping `type` and `rel` as `image/png` and `shortcut icon` consistent:
```
export const updateFavicon = (active = false): void => {
  const linkEl: HTMLLinkElement | null = document.querySelector("link[rel*='shortcut icon']");
  if (!linkEl) return;

  const suffixDev = process.env.IS_DEV ? '-dev' : '';
  const suffixActive = active ? '-active' : '';
  linkEl.href = `/favicons/favicon${suffixDev}${suffixActive}.png`;
};
```

Reply:
I think we can just switch to the `png` as a default favicon and just reserve `favicon.ico` as something crawlers can look for and the very first default favicon.